src/components/Articles.jsx

```diff
@@ -0,0 +1,62 @@
+import {useEffect, useState } from "react"
+import { getArticles } from '../api'
+
+const Articles = () => {
+    
+    const [articlesList, setArticlesList] = useState([])
+    const [sortBy, setSortBy] = useState('')
+    const [orderBy, setOrderBy] = useState('')
+    useEffect(()=>{
+        getArticles(sortBy, orderBy).then((res)=>{
+            
+            setArticlesList(res.articles)
+        })
+    }, [sortBy, orderBy])
+
```

Comment:
It would be really nice, if the user could get some visual feedback in form of a loading state - it can get quite confusing to be greeted by a blank page only showing the header :)